packages/vscode-extension/package.nls.json

```diff
@@ -256,6 +258,13 @@
     "teamstoolkit.localDebug.failedCheckers": "Unable to check: [%s].",
     "teamstoolkit.handlers.askInstallOfficeAddinDependency": "Install dependencies for Office Add-in?",
     "teamstoolkit.handlers.installOfficeAddinDependencyCancelled": "Dependency installation is canceled, but you can install dependencies manually by clicking the 'Development - Check and Install Dependencies' button on the left side.",
+    "teamstoolkit.handlers.askInstallCopilot": "To start with Github Copilot, we recommend you to install the extenstion first.",
+    "teamstoolkit.handlers.askInstallCopilot.install": "Install",
+    "teamstoolkit.handlers.askInstallCopilot.cancel": "Cancel",
+    "teamstoolkit.handlers.installExtension.output": "You need to install %s first.",
+    "teamstoolkit.handlers.installCopilotError": "Cannot install Github Copilot Chat. Please install it from extension marketplace and try again.",
+    "teamstoolkit.handlers.chatTeamsAgentError": "Cannot focus Github Copilot Chat automatically. Please open Github Copilot Chat and start with \"@teams\"",
+    "teamstoolkit.handlers.verifyCopilotExtensionError": "Failed to verify Github Copilot Chat. Please install it manually and try again.",
```

Comment:
Proposed: Unable to verify Github Copilot Chat. Install it manually and try again.